Add support for the various file systems as found in `sheevaplug.h'.
In addition, increase the malloc area size to allow compilation of UbiFS.

--- u-boot/include/configs/guruplug.h	1970-01-01 01:00:01.000000000 +0100
+++ u-boot/include/configs/guruplug.h	2010-09-29 18:59:52.000000000 +0200
@@ -145,7 +145,7 @@
 /*
  * Size of malloc() pool
  */
-#define CONFIG_SYS_MALLOC_LEN	(1024 * 128) /* 128kB for malloc() */
+#define CONFIG_SYS_MALLOC_LEN	(1024 * 1024) /* 1 MiB for malloc() */
 /* size in bytes reserved for initial data */
 #define CONFIG_SYS_GBL_DATA_SIZE	128
 
@@ -195,4 +195,18 @@
 
 #define CONFIG_SYS_ALT_MEMTEST
 
+/*
+ * File system
+ */
+#define CONFIG_CMD_EXT2
+#define CONFIG_CMD_FAT
+#define CONFIG_CMD_JFFS2
+#define CONFIG_CMD_UBI
+#define CONFIG_CMD_UBIFS
+#define CONFIG_RBTREE
+#define CONFIG_MTD_DEVICE               /* needed for mtdparts commands */
+#define CONFIG_MTD_PARTITIONS
+#define CONFIG_CMD_MTDPARTS
+#define CONFIG_LZO
+
 #endif /* _CONFIG_GURUPLUG_H */

